src/clarity/clarityValue.ts

```diff
@@ -45,6 +48,46 @@ export type ClarityValue =
   | ListCV
   | TupleCV;
 
+export function cvToString(val: ClarityValue, encoding: 'tryAscii' | 'hex' = 'tryAscii'): string {
+  switch (val.type) {
+    case ClarityType.BoolTrue:
+      return 'true';
+    case ClarityType.BoolFalse:
+      return 'false';
+    case ClarityType.Int:
+      return val.value.fromTwos(CLARITY_INT_SIZE).toString();
+    case ClarityType.UInt:
+      return val.value.toString();
+    case ClarityType.Buffer:
+      if (encoding === 'tryAscii') {
+        try {
+          const ascii = val.buffer.toString('ascii');
+          if (/[ -~]/.test(ascii)) {
+            return `"${ascii}"`;
+          }
+        } catch (e) {}
```

Comment:
Is this going to fail silently vs throwing an error for the other value types?

Reply:
I'd say so, we should be throwing the error

Reply:
Just removed this as it turns out that `Buffer.toString()` doesn't throw anyway